README.md

```diff
@@ -0,0 +1,43 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TODO List
+
+> TODO List you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server
+
+<!-- ## Live preview
+
+Live preview [link](https://isaacpitwa.github.io/my-portfolio/) -->
+
+## Built With
+
+- Webpack
+- HTML & CSS
+- GitFlow
+- Linters
+- Documented Work
+
```

Comment:
- [ ] Please it will be nice to add in your `readme file` a `setup section` with `steps` to clone and run your app locally, as this will be of help to none technical users who would like to know how your app works.